### PR TITLE
Assignment4-Javier Montoya

### PR DESCRIPTION
**Pull request by Javiguti**
Added tasks 6 and 7

### CODE REVIEW

**Comment by dgarijo** [comment]
Task7 is incomplete.

**Comment by Javiguti** [comment]
added task7

File: Assignment4/Javiguti-22A247/task6.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/Javiguti-22A247/task6.py:10]: won't work in Python. This is notebook magic

File: Assignment4/Javiguti-22A247/task7.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+for r in g.query(q1):
+  print(r)
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**"""
+
+for s, p, o in g.triples((None, RDF.type, ns.Person)):
+  print(f"{s}")
+for s,p,o in g.triples((None, RDFS.subClassOf, ns.Person)):
+  for s2,p2,o2 in g.triples((None, RDF.type, subClass)):
```
Comment (dgarijo) [inline Assignment4/Javiguti-22A247/task7.py:43]: where is the subCalss variable coming from?

Comment (dgarijo) [reply Assignment4/Javiguti-22A247/task7.py:43]: You have not run this script/notebook...

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Comment by Javiguti** [comment]
Hey, Im sorry, I did not save before the final changes and uploaded the incomplete one. Sorry! it should work now

**Review by dgarijo — APPROVED** [review]

**Pull request merged**